website/content/en/docs/helm/quickstart.md

```diff
@@ -231,83 +230,113 @@ using a bit of setup from other `operator-sdk` commands. Check out the OLM integ
+nginx-operator-metrics                              ClusterIP   10.107.164.77   <none>        8383/TCP,8686/TCP   4m10s
+nginx-sample                                        ClusterIP   10.96.152.76    <none>        8080/TCP            2m51s
 ```
 
 ### Update the replicaCount and remove the port
 
 Change the `spec.replicaCount` field from 2 to 3, remove the `spec.service`
-field, and apply the change:
+field:
 
 ```sh
-$ cat deploy/crds/example.com_v1alpha1_nginx_cr.yaml
-apiVersion: "example.com/v1alpha1"
-kind: "Nginx"
-metadata:
-  name: "example-nginx"
-spec:
-  replicaCount: 3
+$ cat config/samples/example_v1alpha1_nginx.yaml
+  apiVersion: example.com/v1alpha1
+  kind: Nginx
+  metadata:
+    name: nginx-sample
+  spec:
+    # Default values copied from <project_dir>/helm-charts/nginx/values.yaml
+    ...
+    replicaCount: 3
+    resources: {}
+    securityContext: {}
+    serviceAccount:
+      annotations: {}
+      create: true
+      name: ""
+    tolerations: []
+```
 
-$ kubectl apply -f deploy/crds/example.com_v1alpha1_nginx_cr.yaml
+And apply the change:
+
+```sh
+$ kubectl apply -f config/samples/example_v1alpha1_nginx.yaml -n nginx-operator-system
 ```
 
 Confirm that the operator changes the deployment size:
 
 ```sh
-$ kubectl get deployment
-NAME                                           DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
-example-nginx-b9phnoz9spckcrua7ihrbkrt1        3         3         3            3           1m
+$ kubectl get deployment -n nginx-operator-system
+NAME                                READY   UP-TO-DATE   AVAILABLE   AGE
+nginx-operator-controller-manager   1/1     1            1           9m16s
+nginx-sample                        3/3     3            3           7m29s
 ```
 
 Check that the service port is set to the default (`80`):
 
 ```sh
-$ kubectl get service
-NAME                                      TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)  AGE
-example-nginx-b9phnoz9spckcrua7ihrbkrt1   ClusterIP   10.96.26.3   <none>        80/TCP   1m
+$ kubectl get service  -n nginx-operator-system
+NAME                                                TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)             AGE
+nginx-operator-controller-manager-metrics-service   ClusterIP   10.99.175.79    <none>        8443/TCP            9m42s
+nginx-operator-metrics                              ClusterIP   10.107.164.77   <none>        8383/TCP,8686/TCP   9m13s
+nginx-sample                                        ClusterIP   10.96.152.76    <none>        80/TCP              7m54s
+```
+
+### Troubleshooting
+
+Use the following command to check the operator logs.
+
+```sh
+$ kubectl logs deployment.apps/nginx-operator-controller-manager  -n nginx-operator-system -c manager
+```
+
+Use the following command to check the CR status and events.
+
+```sh
+$ kubectl describe Nginx -n nginx-operator-system 
```

Comment:
Better to use the full resource.group name in case of conflicts with other kinds of the same name in a different group.

```suggestion
$ kubectl describe nginxes.example.com 
```